aten/src/ATen/native/Normalization.cpp

```diff
@@ -647,6 +666,14 @@ Tensor batch_norm(
   const Tensor& running_var = c10::value_or_else(running_var_opt, [] {return Tensor();});
   return std::get<0>(at::_batch_norm_impl_index(input, weight, bias, running_mean, running_var,
                                                 training, momentum, eps, cudnn_enabled));
+  // TODO: switch to the new stack after the 2 week FC window
```

Comment:
One thing you could try doing for sanity is pushing out a separate PR that does this change, and just makes sure that CI is green (basically a smoke test that the "new" batch norm gives the same dispatching behavior as the old one, according to our test suite)

Reply:
Sounds good. Yes I plan to do that in the next PR